docs/modules/reference/pages/telemetryd/protocols/bmp/bmp.adoc

```diff
@@ -1,18 +1,31 @@
 
 = BGP Monitoring Protocol
+:description: Learn how the BGP Monitoring Protocol (BMP) integration in OpenNMS Horizon/Meridian helps you to see what the Border Gateway Protocol is doing.
 
-The BGP Monitoring Protocol (BMP) provides a convenient interface for obtaining route views.
-The integration in {page-component-title} lets you use these route views, status updates, and statistics for monitoring and management.
+The https://datatracker.ietf.org/doc/html/rfc7854[BGP Monitoring Protocol] (BMP) monitors Border Gateway Protocol (BGP) sessions and provides a convenient interface to monitor BGP routing information on the routing device.
+
+The BMP integration in {page-component-title} lets you use these route views, status updates, and statistics for advanced monitoring and management, providing visibility into what BGP is doing.
+
+== {page-component-title} BMP integration
+
+{page-component-title} leverages https://www.snas.io/[OpenBMP], an application that was developed as a reference implementation for receiving BMP messages from network equipment and presenting the state of the BGP network to the user.
+
+BGP runs on routers and high-end switches.
+Those devices can send BMP data to {page-component-title} directly or through a {page-component-title} Minion collector.
 
 .BMP integration overview
-image::bmp/bmp_integration_overview.png[width=70%]
+image::bmp/bmp_integration_overview_2.png[width=70%]
+
+The {page-component-title} BMP parser accepts BMP connections from router packets using a TCP listener.
+Three adapters in OpenNMS read BMP data, including an inventory list of connected routers and the peers they are established with, and stats and data about what is being exchanged over those peers:
```

Comment:
Since we longer referencing OpenBMP Integration Adapter, Three adapaters might be wrong.

Reply:
Doh! Yes. Thanks for the catch.